pattoo/api/portal/react/src/UsersRow.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+class UsersRow extends React.Component {
+    constructor(props){
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.